stackoverflow-dark.user.css

```diff
@@ -1409,21 +1410,21 @@ regexp("^https?:\/\/((?!area51)\w+\.)*?stackexchange\.com.*") {
   /* prevent animation from breaking while loading  issue #187 */
   @keyframes blinking {
     from {
-      background: rgba(0, 40, 16, .8);
+      background: rgba(/*[[accepted-color-rgb]]*/, .8);
```

Comment:
This breaks animation which is part of problem Ive no idea how to do his without breaking animation either.